rttr-de.po

```diff
@@ -3459,6 +3542,11 @@ msgstr "Wettkampf: %u Minuten"
 msgid "Trade"
 msgstr "Handel"
 
+#: src/figures/nofTradeLeader.cpp:72
+#, c-format
+msgid "Trade caravan with %s %s arrives from player '%s'."
```

Comment:
Hier fehlt die Übersetzung, wobei ich den Englischen Text nicht besonders gut finde (Müsste es nicht eine Art "Trade caraven with %s %s **arrrived** from player '%s'." heißen?

Reply:
Eventuell dann eher "has arrived"

Reply:
ja noch besser

Reply:
Die Nachricht erscheint sobald nur der Karawanenführer am Ziel angekommen ist. Zu dem Zeitpunkt ist die Karawane noch nicht vollständig angekommen, wenn man es genau nimmt. Sie trifft also gerade in diesem Moment ein.

Man könnte die Nachricht auch erst anzeigen, sobald die Karawane vollständig angekommen ist.

Ich hab mal die aktuelle Version übersetzt, wenn ihr das ändern wollt passe ich es natürlich an.
Leider ist es nicht so gut lesbar.

![nachricht](https://user-images.githubusercontent.com/35700046/47987940-1fc3f700-e0e1-11e8-873c-48be211a6011.png)

Eventuell könnte man da statt dem HQ einen Karawanenführer, welchem paar Esel (mit Waren) folgen, als Bild nehmen?

Reply:
Das Bild klingt ganz gut, technisch wäre es aber vermutlich problematisch, oder? Das Bild mit Karawanenführer plus Esel gibt es in dieser Form nicht und eine Nachricht kann immer nur ein Bild bekommen. Man müsste es also dann vorher im Speicher zusammenbasteln, da wir leider die Grafiken des Originals nur nutzen, aber nicht weitergeben können. Wenn man nur den Scout dort abbildet, müsste es besser passen, oder? Alternativ könnte man das Bild weglassen. Aber das sollten Leute beraten, die mehr Sinn von Ästhetik haben als ich... ;-)

Reply:
Genau über dieses Thema habe ich gerade auch nachgedacht. :)

Darf ich überhaupt Texte von dem Original anpassen?
Manche Hilfetexte welche übernommen wurden, passen nicht mehr zur aktuellen Version.
Die müsste man dann komplett neu formulieren?

Reply:
Also ich würde einfach einen Esel nehmen der die Ware in der Tasche hat die angekommen ist. Als Text würde doch reichen "8 Schmelztiegel erhalten" oder "Ein Handel mit %n wurde abgeschlossen"

Reply:
Prinzipiell alles möglich. Wir packen ja z.B. auch den Schatten und das Bild zusammen. Ich arbeite gerade an einem besseren Resourcenverwaltungssystem, aber solche zusammengesetzten Sachen sind echt schwierig. @MarcusSt Wenn du mal Intresse und Zeit hast... vieleicht hast du ja noch eine Idee.

Für jetzt würde ich einfach einen Esel (ohne was) nehmen. Wenn wir das mit dem Scout-Leader mergen, dann eher den.

Reply:
I think the shorter version is better: "8 Schmelztiegel von 'TEST' erhalten", da ist alles wichtige drin. Oder "...angekommen"